packages/bpk-react-utils/src/deprecated.js

```diff
@@ -26,7 +26,7 @@ const deprecated = (propType: PropType, alternativeSuggestion: string) => (
   ...rest: [any]
 ) => {
   if (props[propName] != null) {
-    const message = `"${propName}" property of "${componentName}" has been deprecated. ${alternativeSuggestion}`;
+    const message = `Warning: "${propName}" property of "${componentName}" has been deprecated. ${alternativeSuggestion}`;
```

Comment:
So that Danger can recognise and report this

Reply:
🙇